torch/lib/c10d/test/FileStoreTest.cpp

```diff
@@ -46,7 +46,7 @@ void testHelper(const std::string prefix = "") {
 
   // Perform get on new instance
   {
-    c10d::FileStore fileStore(path);
+    c10d::FileStore fileStore(path, 1);
```

Comment:
Should be 2, same for case above. This tests set/get across two instances. With 1, the file will be gone.